pkg/config/v2/errors/errors.go

```diff
@@ -36,6 +41,10 @@ type InvalidJsonError struct {
 	EnvironmentDetails EnvironmentDetails
 	WrappedError       error
 }
+type RespError struct {
```

Comment:
This file is intended for errors that relate to configurations. 
Please add the new error to a dedicated errors file in either the rest/ or client/ package (not sure where it fits better)

Reply:
I'll go with Client since it's where it's thrown

Reply:
Done